package.json

```diff
@@ -48,6 +48,7 @@
     "node": ">=18.0.0 <19.0.0"
   },
   "dependencies": {
-    "@wordpress/icons": "^10.6.0"
+    "@wordpress/icons": "^10.6.0",
+    "webpack-dev-server": "^5.2.0"
```

Comment:
just wondering why webpack-dev-server got added, is this to resolve a nested dependency somewhere?